plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -63,20 +63,18 @@
   wiki_enabled:
     description:
       - If an wiki for this project should be available or not.
-      - Possible values are true and false.
     type: bool
     default: yes
   snippets_enabled:
     description:
       - If creating snippets should be available or not.
-      - Possible values are true and false.
     type: bool
     default: yes
   visibility:
     description:
-      - Private. Project access must be granted explicitly for each user.
-      - Internal. The project can be cloned by any logged in user.
-      - Public. The project can be cloned without any authentication.
+      - C(Private) Project access must be granted explicitly for each user.
+      - C(Internal) The project can be cloned by any logged in user.
+      - C(Public) The project can be cloned without any authentication.
```

Comment:
```suggestion
      - C(private) Project access must be granted explicitly for each user.
      - C(internal) The project can be cloned by any logged in user.
      - C(public) The project can be cloned without any authentication.
```
The values are lower-case according to the `choices` below.